src/app/app.component.html

```diff
@@ -3,13 +3,13 @@
 
     <div class="m-0 p-0">
       <nav class="nav flex-column">
-        <a class="nav-brand mx-auto mb-1" routerLink=""><img src="assets/images/logo/airsonic-light-350x100.png"></a>
+        <a class="nav-brand mx-auto mb-1" routerLink=""><img src="./../assets/images/logo/airsonic-light-350x100.png"></a>
```

Comment:
Why adding those relative path access ?
With what are you developing ? angular-cli ?